ansible-role-requirements.yml

```diff
@@ -10,3 +10,7 @@
   scm: git
   src: https://github.com/ceph/ansible-ceph-osd.git
   version: 434a13f4d3ee3f60c410052d2670d5dfea034bf8
+- name: openstack_hosts
```

Comment:
Let's add a comment here detailing why we're forking the repo and include the JIRA number as well so we can reference this change easily in the future.

Reply:
Other than that, it looks good, points to the https://github.com/rcbops/openstack-ansible-openstack_hosts/tree/leapupgrade-version which is the change on top of the newton code.

Reply:
Yeah, gonna add comment to explain the separated role.